pyaedt/modules/Material.py

```diff
@@ -858,31 +992,31 @@ def is_conductor(self, threshold=100000):
 
     @aedt_exception_handler
     def is_dielectric(self, threshold=100000):
-        """Check if material is dielectric
+        """Check if the material is a dielectric.
 
         Parameters
         ----------
-        threshold : float
-            Threshold to define if material is dielectric. Default conductivitivy value is ``100000``
+        threshold : float, optional
+            Threshold to define if a material is a dielectric. The
```

Comment:
```suggestion
            Threshold to define if a material is dielectric. The
```

Reply:
I did this as I consider dielectric like "conductive" (i.e. an adjective).  Feel free to ignore suggestion, but this seems to be in agreement with Massimo's work.